pkg/invariants/disruptionimageregistry/invariant.go

```diff
@@ -47,6 +50,19 @@ func NewRecordAvailabilityOnly() invariants.InvariantTest {
 func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	var err error
 
+	coreClient, err := e2e.LoadClientset(true)
+	if err != nil {
+		return fmt.Errorf("unable to load client set: %v", err)
+	}
+	isMicroShift, err := exutil.IsMicroShiftCluster(coreClient)
```

Comment:
This isn't actually microshift, right?  It's a check if the imageregistry namespace isn't present.

Reply:
Done